README.md

```diff
@@ -45,7 +45,8 @@ This builds the plugin in `/build/rootfs`
 
 ###  Task manifest
 User need to provide following parameters in configuration for publisher:
-- `address` -  IP address with port number for RabbitMQ host,
+- `uri` -  AMQP URI scheme for connecting to server. URI supports the following form: "user:pass@ip:port/vhost".
+           User, password, and vhost are optional. When not specified, the default values for RabbitMQ will be used.
```

Comment:
Because "guest:guest" does not work for remote hosts without the user explicitly enabling this. There is no default user/password for connecting to remote hosts; users must create user for publishing to remote instances of RMQ.